lib/elasticbeats.py

```diff
@@ -45,6 +45,8 @@ def render_without_context(source, target):
         context.update({'has_containers': True})
     if path.isdir('/var/lib/docker/containers'):
         context.update({'has_docker': True})
+    if context['kube_logs'] and path.isfile('/root/.kube/config'):
```

Comment:
The `kube_logs` config only exists on the filebeat charm, right?

Will `context['kube_logs']` result in a `KeyError` on other beats charms like topbeat?

Reply:
Yikes, you're right.  Nice catch.